pkg/operation/botanist/controlplane.go

```diff
@@ -289,6 +297,10 @@ func (b *Botanist) HibernateControlPlane(ctx context.Context) error {
 		}
 	}
 
+	if err := c.Delete(ctx, &corev1.Service{ObjectMeta: metav1.ObjectMeta{Name: v1beta1constants.DeploymentNameKubeAPIServer, Namespace: b.Shoot.SeedNamespace}}, kubernetes.DefaultDeleteOptions...); client.IgnoreNotFound(err) != nil {
```

Comment:
Can we please only do this if the seed does not disable DNS? Otherwise, most kubeconfigs are invalided and we can't refresh them.

Reply:
Nice catch!